templates/qpidd.conf.erb

```diff
@@ -12,27 +12,27 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-# http://www.apache.org/licenses/LICENSE-2.0 
+# http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, 
-# software distributed under the License is distributed on an 
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY 
-# KIND, either express or implied. See the License for the 
-# specific language governing permissions and limitations 
-# under the License. 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
 #
-# Configuration file for qpidd. Entries are of the form: 
-# name=value 
+# Configuration file for qpidd. Entries are of the form:
+# name=value
 #
-# (Note: no spaces on either side of '='). Using default settings: 
-# "qpidd --help" or "man qpidd" for more details. 
-#cluster-mechanism=ANONYMOUS 
-log-enable=error+
-log-to-syslog=yes
-auth=no
-require-encryption=yes
+# (Note: no spaces on either side of '='). Using default settings:
+# "qpidd --help" or "man qpidd" for more details.
+#cluster-mechanism=ANONYMOUS
+log-enable=<%= scope.lookupvar("qpid::log_level") %>
```

Comment:
It's coming out of qpid::config, so it needs to do the scope lookup - but I didn't know about `scope[]`, neat!